guides/common/modules/proc_monitoring-puppet.adoc

```diff
@@ -3,24 +3,26 @@
 [id="Monitoring_Puppet_{context}"]
 = Monitoring Puppet
 
-In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}* and click the name of the selected {SmartProxy}.
-At the *Puppet* tab you can find the following:
-
-* A summary of Puppet events, an overview of latest Puppet runs, and the synchronization status of associated hosts at the *General* sub-tab.
-* A list of Puppet environments at the *Environments* sub-tab.
-
-At the *Puppet CA* tab you can find the following:
-
-* A certificate status overview and the number of autosign entries at the *General* sub-tab.
-* A table of CA certificates associated with the {SmartProxy} at the *Certificates* sub-tab.
-Here you can inspect the certificate expiry data, or cancel the certificate by clicking *Revoke*.
-* A list of autosign entries at the *Autosign entries* sub-tab.
-Here you can create an entry by clicking *New* or delete one by clicking *Delete*.
+[role="_abstract"]
+You can monitor Puppet activity and certificate management through the Smart Proxy interface.
```

Comment:
```suggestion
You can monitor Puppet activity and certificate management through the {SmartProxy} interface.
```

I am a bit unsure. What do you think about "... management for each {SmartProxy}."? Or "... management on a per-{SmartProxy} basis."?